Assets/MixedRealityToolkit/Attributes/MixedRealityExtensionServiceAttribute.cs

```diff
@@ -51,10 +51,12 @@ public virtual BaseMixedRealityProfile DefaultProfile
             get
             {
 #if UNITY_EDITOR
-                string path;
-                if (EditorProjectUtilities.FindRelativeDirectory(PackageFolder, out path))
+                MixedRealityToolkitModuleType moduleType = MixedRealityToolkitFiles.GetModuleFromPackageFolder(PackageFolder);
```

Comment:
Consider doing the following:
- if moduletype is not None, do this new one,
- otherwise fallback to the old way

Reply:
fixed